Flashpoint Release 200 (#35268)
demisto/content: Packs/Flashpoint/Integrations/Ignite/README.md

    | Fetch Type | Whether to fetch the Ignite alerts or the compromised credentials. Would choose "Compromised Credentials" if nothing selected. | False |
    | Severity for Incidents | Set the default severity for the incidents using this instance. | False |
    | Alert Status | Filters the incoming alerts with the provided alert status. | False |
    | Alert Origin | Filters the incoming alerts with the origin of alert. | False |
    | Alert Sources | Filters the incoming alerts with the source of alert. | False |
    | Fetch fresh compromised credentials alerts | Adds the 'is_fresh' flag to compromised credential queries so it only ingests username/password combinations if they haven't been seen before. | False |
    | Source Reliability | Reliability of the source providing the intelligence data. | False |
    | Create relationships | Create relationships between indicators as part of Enrichment. | False |
    | Trust any certificate (not secure) |  | False |
    | Use system proxy settings |  | False |

6. Click **Test** to validate the URLs, token, and connection.

> ## Configuration for fetching Ignite Alerts as an XSOAR Incident

Reviewer: ```suggestion
## Configuration for fetching Ignite Alerts as a Cortex XSOAR Incident
```

> 1. Select Fetches incidents.

Reviewer: ```suggestion
1. Select **Fetches incidents**.
```

2. Under Classifier, select "N/A".
3. Under Incident type, select "Ignite Alert".
4. Under Mapper (incoming), select "Ignite Alert - Incoming Mapper" for default mapping.

> 5. Enter the connection parameters. (Server URL, API key)

Reviewer: ```suggestion
5. Enter the connection parameters (Server URL, API key).
```

6. Under the Fetch Type, select "Alerts".

> 7. Select "Alert Status" based on your requirement to filter the alerts.By default, it will fetch all status of alerts.
> 8. Select "Alert Origin" based on your requirement to filter the alerts.By default, it will fetch all origins of alerts.
> 9. Select "Alert Sources" based on your requirement to filter the alerts.By default, it will fetch all sources of alerts.

Reviewer: ```suggestion
7. Select "Alert Status" based on your requirement to filter the alerts. By default, it will fetch all status of alerts.
8. Select "Alert Origin" based on your requirement to filter the alerts. By default, it will fetch all origins of alerts.
9. Select "Alert Sources" based on your requirement to filter the alerts. By default, it will fetch all sources of alerts.
```

10. Select "Severity for Incidents" based on your requirement to set the default severity for the incidents.By default, it will set the Unknown severity for all incidents.

> 11. Update "First fetch time interval" & "Max Fetch Count" based on your requirement.

Reviewer: ```suggestion
11. Update "First fetch time interval" and "Max Fetch Count" based on your requirements.
```

> ## Configuration for fetching Ignite Compromised Credentials as an XSOAR Incident

Reviewer: ```suggestion
## Configuration for fetching Ignite Compromised Credentials as a Cortex XSOAR Incident
```

> 1. Select Fetches incidents.

Reviewer: ```suggestion
1. Select **Fetches incidents**.
```

2. Under Classifier, select "N/A".
3. Under Incident type, select "Flashpoint Compromised Credentials".
4. Under Mapper (incoming), select "Flashpoint Compromised Credentials - Incoming Mapper" for default mapping.

> 5. Enter the connection parameters. (Server URL, API key)

Reviewer: ```suggestion
5. Enter the connection parameters (Server URL, API key).
```

6. Under the Fetch Type, select "Compromised Credentials".
7. Select "Fetch fresh compromised credentials alerts" so that it only ingests username/password combinations if they haven't been seen before.

> 8. Update "First fetch time interval" & "Max Fetch Count" based on your requirement.

Reviewer: ```suggestion
8. Update "First fetch time interval" and "Max Fetch Count" based on your requirements.
```
